tests/vmi_tpm_test.go

```diff
@@ -85,3 +101,182 @@ var _ = Describe("[sig-compute]vTPM", decorators.SigCompute, func() {
 		})
 	})
 })
+
+var _ = Describe("[sig-storage]vTPM", func() {
+	var virtClient kubecli.KubevirtClient
+	var err error
+
+	BeforeEach(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("with persistent TPM VM option enabled", func() {
+		restartVM := func(vm *v1.VirtualMachine) {
+			By("Stopping the VM")
+			err = virtClient.VirtualMachine(vm.Namespace).Stop(context.Background(), vm.Name, &v1.StopOptions{})
+			ExpectWithOffset(1, err).ToNot(HaveOccurred())
+			EventuallyWithOffset(1, func() error {
+				_, err = virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &k8smetav1.GetOptions{})
+				return err
+			}, 300*time.Second, 1*time.Second).ShouldNot(Succeed())
+
+			By("Starting the VM")
+			err = virtClient.VirtualMachine(vm.Namespace).Start(context.Background(), vm.Name, &v1.StartOptions{})
+			ExpectWithOffset(1, err).ToNot(HaveOccurred())
+			var vmi *v1.VirtualMachineInstance
+			EventuallyWithOffset(1, func() error {
+				vmi, err = virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &k8smetav1.GetOptions{})
+				return err
+			}, 300*time.Second, 1*time.Second).Should(Succeed())
+			libwait.WaitForSuccessfulVMIStartWithTimeout(vmi, 60)
+
+			By("Logging in as root")
+			err = console.LoginToFedora(vmi)
+			Expect(err).ToNot(HaveOccurred())
+		}
+
+		migrateVMI := func(vmi *v1.VirtualMachineInstance) {
+			By("Migrating the VMI")
+			checks.SkipIfMigrationIsNotPossible()
```

Comment:
this `Skip` can be done at the beginning of the test to save some time and resources

Reply:
Got it! Thanks for the explanation :+1: